components/adv-post-form.js

```diff
@@ -198,7 +198,7 @@ export default function AdvPostForm ({ children, item, sub, storageKeyPrefix })
       for (let i = 0; i < MAX_FORWARDS; i++) {
         ['nym', 'pct'].forEach(key => {
           const value = window.localStorage.getItem(`${storageKeyPrefix}-forward[${i}].${key}`)
-          if (value) {
+          if (value !== undefined && value !== null) {
```

Comment:
This is a bug fix unrelated to this PR, but I noticed it while testing so I fixed it.